Lesson_8_Dictionary/Homework_8.py

```diff
@@ -0,0 +1,38 @@
+# Task_1 Hello Mr. key
+roles = { 'admin' : ['Evgeniy', 'Dmitry', 'asd'], 'maintainer' : ["Marina", "Sergey"],
+          'manager' : ["Anastasia", "Dmitry", 'asd'], 'developer': ["Evgeniy"]}
+name = input('What is yor name?')
+name_roles = []
+for key, value in roles.items():
+    if name in value:
+        name_roles.append(key)
+if name_roles:
+    print(f'Hello, {' & ' .join(name_roles)}!')
+else:
+    print('Hello, Guest!')
```

Comment:
Working) but can be better
![image](https://github.com/EvgeniyPon/Hillel_Lessons/assets/51053483/8056feb3-8406-4ad9-bcf1-b5e7f39da767)

Reply:
thx for checking & comment! 
I have been started from this way,  but that code is not consider about more than one roles and case of non title name